packages/framework/src/Model/Order/OrderFacade.php

```diff
@@ -161,6 +167,7 @@ public function __construct(
         OrderNumberSequenceRepository $orderNumberSequenceRepository,
         OrderRepository $orderRepository,
         OrderService $orderService,
+        OrderUrlGenerator $orderUrlGenerator,
```

Comment:
Unfortunatelly I have no idea... Do I have to move it to the end? It will probably cause conflicts and propably some extra effort to put everything together

Reply:
No, you do have to :wink: I was just curious